clay/player.py

```diff
@@ -251,9 +251,11 @@ def _media_state_changed(self, event):
     def _media_end_reached(self, event):
         """
         Called when end of currently played track is reached.
-        Advances to the next track.
+        Increments the play count and advances to the next track.
         """
         assert event
+        track = self.queue.get_current_track()
+        track.inc_playcount()
```

Comment:
This should be called asynchronously, otherwise it will block playing the next track.

Reply:
Yes, and I don't really see why there is a variable assignment if it isn't used more than once.

Reply:
uhh, sorry.
it had been used but that part got removed